### PR TITLE
RunAndGun Multiplayer Compatibility

### PR DESCRIPTION
**Pull request by Thomas107500**
Synced the Run And Gun gizmo, tested with a weapon, no desync

### CODE REVIEW

File: Source/Mods/RunandGun.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+using Multiplayer.API;
+
+namespace Multiplayer.Compat
+{
+    /// <summary>RunAndGun by roolo</summary>
+    /// <see href="https://steamcommunity.com/sharedfiles/filedetails/?id=1204108550"/>
+    [MpCompatFor("roolo.RunAndGun")]
+    class RunandGun
+    {
+        Type type = AccessTools.TypeByName("RunAndGun.Harmony.Pawn_DraftController_GetGizmos_Patch");
```
Comment (notfood) [inline Source/Mods/RunandGun.cs:13]: Can you make Type local?

Comment (Thomas107500) [reply Source/Mods/RunandGun.cs:13]: oh wait it is not local? give me a minute :P

Comment (Thomas107500) [reply Source/Mods/RunandGun.cs]: Done

**Comment by notfood** [comment]
Alright, thanks.

**Pull request merged**